graphql/database/users/logTab.js

```diff
@@ -75,28 +75,28 @@ const logTab = async (userContext, userId, tabId = null, isV4 = true) => {
   } catch (e) {
     throw e
   }
+
   const todayTabCount = getTodayTabCount(user) + 1
   const isValid = isTabValid(todayTabCount, user.lastTabTimestamp)
 
-  // Update the user's counter for max tabs in a day.
-  // If this is the user's first tab today, reset the counter
-  // for the user's "current day" tab count.
-  // If today is also the day of all time max tabs,
-  // update the max tabs day value.
-  const isTodayMax = todayTabCount >= user.maxTabsDay.maxDay.numTabs
```

Comment:
@jedtan we'll still want to calculate and store a user's alltime max tab count.  This is to support legacy users

Reply:
sorry disregard, I see you do it below